src/app/earn/components/quick-stats.tsx

```diff
@@ -1,53 +1,86 @@
+import { useDisclosure } from '@chakra-ui/react'
+
+import { TokenSelector } from '@/app/earn/components/earn-widget/components/base-token-selector'
 import { StatMetric } from '@/app/earn/components/stat-metric'
 import { formatPercentage, formatTvl } from '@/app/products/utils/formatters'
-import { TokenDisplay } from '@/components/token-display'
+import { SelectTokenModal } from '@/components/swap/components/select-token-modal'
+import { useWallet } from '@/lib/hooks/use-wallet'
 import { formatDollarAmount } from '@/lib/utils'
 
 import { useEarnContext } from '../provider'
 
 export function QuickStats() {
-  const { indexToken, isFetchingStats, nav, apy, apy30d, apy7d, tvl } =
-    useEarnContext()
+  const {
+    isOpen: isSelectIndexTokenOpen,
+    onOpen: onOpenSelectIndexToken,
+    onClose: onCloseSelectIndexToken,
+  } = useDisclosure()
+  const {
+    indexToken,
+    indexTokens,
+    isFetchingStats,
+    nav,
+    apy,
+    apy30d,
+    apy7d,
+    tvl,
+    onSelectIndexToken,
+  } = useEarnContext()
+  const { address } = useWallet()
 
   return (
-    <div className='border-ic-gray-200 divide-ic-gray-200 flex w-full items-center justify-between divide-x rounded-lg border bg-[#F7F8F8]'>
-      <div className='flex w-1/2 items-center justify-between px-4 py-2 sm:py-3 md:px-8 md:py-4'>
-        <div className='flex'>
-          <TokenDisplay smHideLabel token={indexToken} />
+    <div className='border-ic-gray-200 flex w-full items-center justify-between rounded-lg border bg-[linear-gradient(180deg,#FCFFFF,#F7F8F8)]'>
```

Comment:
wonder if this could use one of the linear graadient presets: https://tailwindcss.com/docs/background-image#linear-gradients